test/del-test.js

```diff
@@ -1,5 +1,5 @@
 const test = require('tape')
-const leveldown = require('..')
-const abstract = require('abstract-leveldown/abstract/del-test')
+const testCommon = require('./common')
+const abstract = require('abstract-leveldown/test/del-test')
 
-abstract.all(leveldown, test)
+abstract.all(testCommon.factory, test)
```

Comment:
A good thing here is that `testCommon.factory` is already prepared, so once we change `abstract-leveldown` we only need to flip arguments in `leveldown`.